lenskit-slim/src/main/java/org/lenskit/slim/LinearRegression.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * LensKit, an open source recommender systems toolkit.
+ * Copyright 2010-2016 LensKit Contributors.  See CONTRIBUTORS.md.
+ * Work on LensKit has been funded by the National Science Foundation under
+ * grants IIS 05-34939, 08-08692, 08-12148, and 10-17697.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package org.lenskit.slim;
+
+
+import org.grouplens.grapht.annotation.DefaultImplementation;
+
+import javax.inject.Qualifier;
+import java.lang.annotation.*;
+
+
+@Documented
+@Qualifier
+@DefaultImplementation(CovarianceUpdateCoordDestLinearRegression.class)
+@Target({ElementType.TYPE, ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface LinearRegression {
```

Comment:
This interface doesn't seem to be adding anything - qualifiers would get applied to injection points, not to injectable interfaces, so it doesn't seem to be used.